examples/testing-dom__csv-table/cypress/integration/spec.js

```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress" />
+const neatCSV = require('neat-csv')
+
+it('has table rows for each row of CSV file', () => {
+  cy.visit('index.html')
+  cy.readFile('records.csv')
+  .then(neatCSV) // converts text into list of objects
+  /* eslint-disable-next-line no-console */
+  .then(console.table) // convenient method for printing list of objects in DevTools console
+  .then((records) => {
+    cy.get('table tr').should(($rows) => {
+      // go through each row and confirm it shows the right information from CSV
+      $rows.each((k, row) => {
+        const record = records[k]
+        const cells = row.children
```

Comment:
Would probably update all these variables with `$` to clarify that they are jquery references.
```suggestion
        const $cells = $row.children
```

Reply:
Went ahead and updated myself.